### PR TITLE
Create modal.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Modal-screenshot

### CODE REVIEW

File: Components/modal.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
```
Comment (junaidshaikh-js) [inline Components/modal.css:11]: background-color repeated.

**Review by junaidshaikh-js — COMMENTED** [review]
Good work!

File: Components/modal.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
```
Comment (Hamdrive) [inline Components/modal.css]: Use `rems` for larger values, even better to use CSS variables

File: Components/modal.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal-content {
+  background-color: #fefefe;
```
Comment (Hamdrive) [inline Components/modal.css]: Define colors in general CSS files, import variables and use them instead

**Review by Hamdrive — COMMENTED** [review]
Highly suggest to change the approach for using colors and units, preferably store them in a general CSS file and import and use CSS variables instead, good industry practice.

File: Components/modal.css (hunk 2 above)
Comment (Radheshyam1234) [reply Components/modal.css]: Done

File: Components/modal.css (hunk 3 above)
Comment (Radheshyam1234) [reply Components/modal.css]: Done

File: Components/modal.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/modal.css:11]: Done

**Pull request merged**